cmd/ims/main.go

```diff
@@ -9,14 +9,17 @@ import (
 	"github.com/sirupsen/logrus"
 	flag "github.com/spf13/pflag"
 
+	"github.com/omc-college/management-system/pkg/config"
 	"github.com/omc-college/management-system/pkg/ims/api/routers"
 	"github.com/omc-college/management-system/pkg/ims/service"
-	"github.com/omc-college/management-system/pkg/config"
 )
 
 func main() {
 	var conf Config
 	var err error
+	var expirationTime time.Time
+
+	expirationTime = time.Now()
```

Comment:
please move it to config, and change type to the duration. Use 24h as default value.